src/strings.ts

```diff
@@ -148,6 +148,7 @@ This is caused by Twitter API downtime or a new bug. Try again in a little while
   ERROR_API_FAIL: 'Tweet failed to load due to an API error :(',
   ERROR_PRIVATE: `Due to Twitter API changes, some NSFW Tweets are currently being blocked. We are currently looking into a workaround. 🙏`,
   ERROR_TWEET_NOT_FOUND: `Sorry, that Tweet doesn't exist :(`,
+  ERROR_USER_NOT_FOUND: `Sorry, that User doesn't exist :(`,
```

Comment:
User doesn't need capitalized here in the string. Tweet is only capitalized because Twitter often does that and it's what I ended up standardizing on.